app/common/containers/pages/InnmDosagesCreatePage/redux.js

```diff
@@ -7,6 +7,7 @@ export const onSubmit = v => (dispatch) => {
     name: v.name,
     ingredients: (v.ingredients || []).map(i => ({
       id: i.id.name,
+      is_primary: v.is_primary,
```

Comment:
Try to avoid usage one-letter variables (oneliners is an exception to the rule). In this case, we can use function parameter destructuring.